geopandas/_compat.py

```diff
@@ -138,23 +141,12 @@ def set_use_pygeos(val=None):
         except ImportError:
             raise ImportError(INSTALL_PYGEOS_ERROR)
 
-    if USE_PYGEOS and env_use_pygeos is None and SHAPELY_GE_20:
+    if USE_PYGEOS:
         warnings.warn(
-            "Shapely 2.0 is installed, but because PyGEOS is also installed, "
-            "GeoPandas still uses PyGEOS by default. However, starting with "
-            "version 0.14, the default will switch to Shapely. To force to "
-            "use Shapely 2.0 now, you can either uninstall PyGEOS or set the "
-            "environment variable USE_PYGEOS=0. "
-            "You can do this before starting the Python process, or in your code "
-            "before importing geopandas:"
-            "\n\nimport os\nos.environ['USE_PYGEOS'] = '0'\nimport geopandas\n\n"
-            "In the next release, GeoPandas will switch to using Shapely by default, "
-            "even if PyGEOS is installed. If you only have PyGEOS installed to "
-            "get speed-ups, this switch should be smooth. However, if you are "
-            "using PyGEOS directly (calling PyGEOS functions on geometries "
-            "from GeoPandas), this will then stop working and you are encouraged to "
-            "migrate from PyGEOS to Shapely 2.0 "
-            "(https://shapely.readthedocs.io/en/latest/migration_pygeos.html).",
+            "GeoPandas is set to use PyGEOS over Shapely. PyGEOS support is deprecated"
```

Comment:
```suggestion
            "GeoPandas is set to use PyGEOS over Shapely when shapely < 2 is installed. PyGEOS support is deprecated"
```

Reply:
> GeoPandas is set to use PyGEOS over Shapely when shapely < 2 is installed. PyGEOS support is deprecated

This isn't quite correct, since the user may set `USE_PYGEOS=True` even if Shapely >= 2.0 is installed.